questionnaires/chair_rises/chair_rises_armt.json

```diff
@@ -0,0 +1,403 @@
+                "label": "Hard"
+            },
+            {
+                "code": "6",
+                "label": "6"
+            },
+            {
+                "code": "7",
+                "label": "Very Hard"
+            },
+            {
+                "code": "8",
+                "label": "8"
+            },
+            {
+                "code": "9",
+                "label": "9"
+            },
+            {
+                "code": "10",
+                "label": "Very very hard, maximal effort"
+            }
+        ],
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[were_you_able_to_complete] = '1'",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "range-info",
+        "evaluated_logic": "responses['were_you_able_to_complete'] ==  '1'"
+    },
+    {
+        "field_name": "chair_rises_completed_10",
+        "form_name": "chair_rises",
+        "section_header": "",
+        "field_type": "yesno",
+        "field_label": "Where you able to complete 10 chair rises?",
+        "select_choices_or_calculations": "",
+        "field_note": "",
+        "text_validation_type_or_show_slider_number": "",
+        "text_validation_min": "",
+        "text_validation_max": "",
+        "identifier": "",
+        "branching_logic": "[were_you_able_to_complete] = '1'",
+        "required_field": "",
+        "custom_alignment": "",
+        "question_number": "",
+        "matrix_group_name": "",
+        "matrix_ranking": "",
+        "field_annotation": "",
+        "evaluated_logic": "responses['were_you_able_to_complete'] ==  '1'"
+    },
+    {
+        "field_name": "how_many_chair_rises_were",
+        "form_name": "chair_rises",
+        "section_header": "",
+        "field_type": "radio",
+        "field_label": "How many chair rises were you able to complete?",
+        "select_choices_or_calculations": [
+            {
+                "code": "1",
+                "label": "1"
+            },
+            {
+                "code": "2",
+                "label": "2"
+            },
+            {
+                "code": "3",
+                "label": "3"
+            },
+            {
+                "code": "4",
+                "label": "4"
+            },
+            {
+                "code": "5",
+                "label": "5"
+            },
+            {
+                "code": "6",
+                "label": "6"
+            },
+            {
+                "code": "7",
+                "label": "7"
+            },
+            {
+                "code": "8",
+                "label": "8"
+            },
+            {
+                "code": "9",
+                "label": "9"
```

Comment:
should the maximum number of chair-rises be 10?

Reply:
I think the user will be prompted this question if the user did not complete the 10 chair rises?